app/main.py

```diff
@@ -1,13 +1,28 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "errors": [
+            format_linter_error(error) for error in errors
+        ],
```

Comment:
this key-value pair could go in one line, since it is not too long. It is up to you to decide whether to change it or not in this case. So I will not reject if you decide leave it as it is.